omero/users/cli/installation.rst

```diff
@@ -15,17 +15,23 @@ to install it first, see `miniconda <https://docs.conda.io/en/latest/miniconda.h
 
 .. Note:: On Ubuntu 20.04, you may need to install ``libssl-dev`` before installing the |CLI|.
 
+Before installing ``omero-py``, we recommend to install the `Zeroc IcePy 3.6 <https://zeroc.com/ice/downloads/3.6>`_ Python bindings.
 
-To install ``omero-py`` using conda (preferred)::
+Our commercial partner `Glencoe Software <https://www.glencoesoftware.com>`_ has produced several Python wheels to install the Ice-Python bindings depending on the desired Python version and the operating system. Please visit the `Ice binaries for omero <https://www.glencoesoftware.com/blog/2023/12/08/ice-binaries-for-omero.html>`_ page to find the full URLs to the wheels that are used in the examples below.
 
-    conda create -n myenv -c conda-forge python=3.8 omero-py
+To install omero-py using pip in a conda environment with Python 3.11 on Ubuntu 22.04::
+
+    conda create -n myenv python=3.11
     conda activate myenv
+    pip install https://github.com/glencoesoftware/zeroc-ice-py-linux-x86_64/releases/download/20240202/zeroc_ice-3.6.5-cp311-cp311-manylinux_2_28_x86_64.whl
+    pip install omero-py==5.19.5
```

Comment:
Do we need to specify `5.19.5` here? and below?

Reply:
fixed in https://github.com/ome/omero-documentation/pull/2467/commits/b7e7b2ff76f9a5942fb961bd3eabb80cc65ac7c1